src/OpenTelemetry.Instrumentation.AspNetCore/CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 
 ## Unreleased
 
+## 1.5.0-rc.1
+
+Released 2023-Jun-05
+
+* Bumped the package version to `1.5.0-rc.1` to keep its major and minor version
+  in sync with that of the core packages. This would make it more intuitive for
+  users to figure out what version of core packages would work with a given
+  version of this package.
```

Comment:
```suggestion
* Bumped the package version to `1.5.0-rc.1` to keep its major and minor version
  in sync with that of the core packages. This would make it more intuitive for
  users to figure out what version of core packages would work with a given
  version of this package. The pre-releases identifier has also been changed
  from `rc` to `beta` as we believe this more accurately reflects the status of
  this package. We believe the `rc` identifier will be more appropriate as
  semantic conventions reach stability. 
```